keyword_spotting_data_generator/evaluation/extractor/edit_distance_extractor.py

```diff
@@ -60,19 +62,25 @@ def extract_keywords(self, data, window_ms=1000, hop_ms=250):
         selected_window = []
 
         current_start = 0
-        while current_start + window_ms < len(data):
-            window = data[current_start:current_start+window_ms]
+
+        mfcc_audio = self.compute_mfccs(data)
+        whitened = whiten(mfcc_audio)
+        K = 100
+        code_book, distortion = kmeans(whitened, K)
+
+        while current_start + window_ms * (self.sr // 1000) < len(data):
+            window = data[current_start:current_start+window_ms*(self.sr//1000)]
```

Comment:
Yeah sure! I'll make a new variable  `window_size = window_ms*(self.sr//1000)` instead of `window_end = current_start+window_ms*(self.sr//1000)`. 

Since `current_start` is also updated in each iteration, I feel `while current_start+window_size  < len(data)` is more convenient and readable than `window_end = current_start+window_ms*(self.sr//1000); while window_end  < len(data)`.

Reply:
sure, that definitely works!